randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -2163,7 +2167,9 @@ Extra - asset_id: collision_camera_025_B
   * Extra - right_shield_entity: Root:pScenario:rEntitiesLayer:dctSublayers:default:dctActors:doorshieldmissile_001
   * Extra - right_shield_def: actordef:actors/props/doorshieldmissile/charclasses/doorshieldmissile.bmsad
   > Door from Chozo Warrior Arena
-      Before Ghavoran - Ferenia Transport Grapple Box and Can Slide
+      All of the following:
+          Before Ghavoran - Ferenia Transport Grapple Box
+          After Ghavoran - Ferenia Transport Enky Left or Can Slide
```

Comment:
This part really only matters if Slide is ever shuffled again, but I suppose it's fine.